src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -130,18 +130,60 @@ which we divide into several categories:
 
 #### Scanner Hardware
 
+{{ MACROS___make_metadata_table(
+   {
+      "Manufacturer": "REQUIRED",
+      "ManufacturersModelName": "REQUIRED",
+      "Unit": "REQUIRED",
+      "InstitutionName": "RECOMMENDED",
+      "InstitutionAddress": "RECOMMENDED",
+      "InstitutionalDepartmentName": "RECOMMENDED",
+      "BodyPart": "RECOMMENDED",
+   }
+) }}
+
 | **Key name**              | **Requirement level** | **Data type** | **Description**                                                                                                                                                                        |
 | ------------------------- | --------------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Manufacturer              | REQUIRED              | [string][]    | Scanner manufacturer (for example, `"Siemens"`).                                                                                                                                       |
 | ManufacturersModelName    | REQUIRED              | [string][]    | PET scanner model name (for example, `"mMR Biograph"`).                                                                                                                                |
 | Unit                      | REQUIRED              | [string][]    | Unit of the image file; please see BIDS main spec section 6. SI unit for radioactivity (Becquerel) should be used (for example, `"Bq/mL"`). Corresponds to DICOM Tag 0054, 1001 Units. |
 | InstitutionName           | RECOMMENDED           | [string][]    | The name of the institution in charge of the equipment that produced the composite instances. Corresponds to DICOM Tag 0008, 0080 InstitutionName.                                     |
 | InstitutionAddress        | RECOMMENDED           | [string][]    | The address of the institution in charge of the equipment that produced the composite instances. Corresponds to DICOM Tag 0008, 0081 InstitutionAddress.                               |
-| InstitutionDepartmentName | RECOMMENDED           | [string][]    | The department in the institution in charge of the equipment that produced the composite instances. Corresponds to DICOM Tag 0008, 1040 Institutional Department Name.                 |
+| InstitutionalDepartmentName | RECOMMENDED           | [string][]    | The department in the institution in charge of the equipment that produced the composite instances. Corresponds to DICOM Tag 0008, 1040 Institutional Department Name.                 |
```

Comment:
I _think_ this was a typo in the PET BEP?

Reply:
yes, most likely. fix: https://github.com/bids-standard/bids-specification/pull/771